File: dbconnector/src/main/java/ru/rti/kettu/sbjava/model/db/h2/Album.java
package ru.rti.kettu.sbjava.model.db.h2;

import lombok.*;

import javax.persistence.*;
import java.io.Serializable;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table
public class Album implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    @Column(updatable = false, nullable = false)
    private Long id;
    @Column(length = 50)
    private String author;
    @Column(length = 50)
    private String name;
    @Column(length = 4)
    private Integer year;

}

File: consumer/src/main/kotlin/ru/rti/kettu/sbkotlin/model/Album.kt
package ru.rti.kettu.sbkotlin.model

class Album constructor() {
    var id: Long? = null
    var author: String? = null
    var name: String? = null
    var year: Int? = null

    constructor(id: Long?, author: String?, name: String?, year: Int? = 0) : this() {
        this.id = id
        this.author = author
        this.name = name
        this.year = year
    }
}